### DISCUSSION TITLE
Why Grid Template is not working?

### DISCUSSION THREAD

**Discussion by tuhinbhuiyan** [tailpress/tailpress#139] [Q&A]
Just installed tailpress and was trying it out. But I am not sure why Grid Template is not working by default? Even though Utilities has been in import state. Tailwindcss says, 'By default, Tailwind includes grid-template-column utilities for creating basic grids with up to 12 equal width columns. '

**Comment by jeffreyvr** [comment]
I need some additional information:
- Are you running the watcher? `npm run watch`
- In which file are you adding this grid?
